File: examples/python/send.py
import os
import requests

api_url = os.getenv("HYPERMSG_URL", "http://localhost:4000")
token = os.environ["HYPERMSG_TOKEN"]
instance_id = os.environ["HYPERMSG_INSTANCE_ID"]

response = requests.post(
    f"{api_url}/messages/send",
    headers={"Authorization": f"Bearer {token}"},
    json={
        "instanceId": instance_id,
        "to": "+15551234567",
        "body": "Hello from Python",
    },
    timeout=15,
)
response.raise_for_status()
print(response.json())

File: examples/node/send.js
const API_URL = process.env.HYPERMSG_URL || "http://localhost:4000";
const API_TOKEN = process.env.HYPERMSG_TOKEN;
const INSTANCE_ID = process.env.HYPERMSG_INSTANCE_ID;

async function sendText() {
  const response = await fetch(`${API_URL}/messages/send`, {
    method: "POST",
    headers: {
      "content-type": "application/json",
      authorization: `Bearer ${API_TOKEN}`
    },
    body: JSON.stringify({
      instanceId: INSTANCE_ID,
      to: "+15551234567",
      body: "Hello from Node.js"
    })
  });
  console.log(await response.json());
}

sendText().catch((err) => {
  console.error(err);
  process.exit(1);
});
